src/components/attendee/Toggler/Toggler.tsx

```diff
@@ -19,18 +19,19 @@ export const Toggler: React.FC<TogglerProps> = ({
   name,
   ...extraProps
 }) => {
-  const containerClasses = classNames("Toggler", containerClassName);
+  const containerClasses = classNames(CN.toggler, containerClassName);
+
   return (
     <div className={containerClasses}>
-      <label className="switch">
+      <label className={CN.switch}>
         <input
           type="checkbox"
-          checked={toggled}
           name={name}
-          onClick={onChange}
+          onChange={onChange}
+          checked={toggled}
           {...extraProps}
         />
-        <span className="slider round"></span>
+        <span className={`${CN.slider} ${CN.sliderRound}`}></span>
```

Comment:
[NOTE] Usually, to make it more clear the function `classnames` is used (at the expense of a function call), but the above is fine I guess. No need to change.